### ISSUE TITLE
FAILING NVDA SHORTCUTS

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8793]
### Steps to reproduce:
THERE IS A WEB PAGE.
PRESS CTRL + NVDA + F TO OPEN NVDA TEXT SEARCH WINDOW

### Actual behavior:
NOTHING HAPPENS.
### Expected behavior:
NVDA SEARCH WINDOW MUST OPEN.
#### NVDA Installed/portable/running from source:
instal
#### NVDA version:
alfa 16124
#### Windows version:
10 17134.286
#### Name and version of other software in use when reproducing the issue:
chrome 69
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no

[1log.txt](https://github.com/nvaccess/nvda/files/2424209/1log.txt)

**Comment by fernando-jose-silva** [comment]
nvda 16123

**Comment by leonardder** [comment]
In that case, would you agree with closing this issue, since it seems you've found the cause of it?

**Comment by fernando-jose-silva** [comment]
Yes thanks.
It seems to be more of a mess of windows.
